workers/api-server/src/http/price_report.rs

```diff
@@ -1,15 +1,31 @@
 //! Groups price reporting API handlers and types
 
 use async_trait::async_trait;
-use external_api::http::price_report::{GetPriceReportRequest, GetPriceReportResponse};
+use common::types::token::{read_token_remap, Token, USDC_TICKER, USDT_TICKER, USD_TICKER};
+use external_api::{
+    http::price_report::{
+        GetPriceReportRequest, GetPriceReportResponse, GetSupportedTokensResponse,
+        GetTokenPricesResponse, TokenPrice,
+    },
+    types::ApiToken,
+    EmptyRequestResponse,
+};
+use futures::{future::join_all, FutureExt};
 use hyper::HeaderMap;
+use itertools::Itertools;
+use job_types::price_reporter::PriceReporterQueue;
 
 use crate::{
     error::{internal_error, ApiServerError},
     router::{QueryParams, TypedHandler, UrlParams},
     worker::ApiServerConfig,
 };
 
+/// Tokens filtered from the supported token endpoint
+const FILTERED_TOKENS: [&str; 2] = [USD_TICKER, USDT_TICKER];
+/// Tokens filtered from the token prices endpoint
+const FILTERED_TOKENS_PRICES: [&str; 3] = [USDT_TICKER, USDT_TICKER, USDC_TICKER];
```

Comment:
I think this was supposed to be:
```suggestion
const FILTERED_TOKENS_PRICES: [&str; 3] = [USD_TICKER, USDT_TICKER, USDC_TICKER];
```

Reply:
good eye